modules/installation-configuration-parameters.adoc

```diff
@@ -462,11 +460,17 @@ ifndef::aws,bare,gcp,ibm-power,ibm-z,azure,ibm-power-vs[]
 |String
 endif::aws,bare,gcp,ibm-power,ibm-z,azure,ibm-power-vs[]
 
-ifdef::aws,bare,azure,gcp[]
+ifdef::gcp,bare[]
+|`compute.architecture`
+|Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. 
+|String
+endif::gcp,bare[]
```

Comment:
@kelbrown20 lines 463-467 state the same thing as lines 469-473. Is there a reason you are repeating the same information twice? Will this module be changed further as part of another PR related to 64-bit ARM? Or is this separated because all installation options for GCP and bare metal support 64-bit ARM?

Outside of my questions, the new content and the your updates to the `ifeval` directives LGTM.

Reply:
Yeah I separated it bc all installation options for GCP and bare metal support 64-bit ARM.

Reply:
> Yeah I separated it bc all installation options for GCP and bare metal support 64-bit ARM.

OK to keep things a bit tidier, I suggest the following in lieu of separating out GCP and bare metal:

ifdef::aws,azure,gcp,bare[]
Determines the instruction set architecture of the machines in the pool. Currently, clusters with varied architectures are not supported. All pools must specify the same architecture. Valid values are `amd64` and `arm64`. 
ifdef::aws,azure[]
Not all installation options support the 64-bit ARM architecture. To verify if your installation option is supported on your platform, see _Supported installation methods for different platforms_ in _Selecting a cluster installation method and preparing it for users_.
endif::aws,azure[]
endif::aws,azure,gcp,bare[]